datahub-web-react/src/app/shared/tags/TagTermGroup.tsx

```diff
@@ -81,6 +95,38 @@ export default function TagTermGroup({
             {domain && (
                 <DomainLink domain={domain} name={entityRegistry.getDisplayName(EntityType.Domain, domain) || ''} />
             )}
+            {uneditableAccess?.roles?.map((role) => {
+                console.log(role);
```

Comment:
Please do not update TagTermGroup directly. Instead, let's use other components for rendering this. TagTermGroup is a very important component that should only deal with .. Tags and Terms.

Reply:
removed the changes as we cannot make changes in the default preview card